src/Controller/ContactFormController.php

```diff
@@ -64,6 +63,15 @@ public function __construct(
         $this->channelContext = $channelContext;
     }
 
+    public function showMessageAction(int $id)
+    {
+        $contactMessages = $this->entityManager->getRepository(ContactFormMessage::class)->find($id);
```

Comment:
You should use `ContactFormRepository` injected in Controller

Reply:
DONE